INSTALL.md

```diff
@@ -210,3 +212,33 @@ You can use the [conda package manager](https://conda.io/) that comes with the
    ```
    conda install gmt -c conda-forge/label/dev
    ```
+
+## FreeBSD
+
+GMT may be installed on FreeBSD using Ports or manually from source.
+
+### Install via Ports
+
+1. If not done already, set up the **Ports Collection** (see https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/ports-using.html)
+
+   ```
+   portsnap fetch
+   ```
+
+and
+
+   ```
+   portsnap extract
+   ```
+
+  If already set up, make sure you're up-to-date:
+
+  ```
+  portsnap fetch update
+  ```
+
+2. ``cd`` into ``/usr/ports/graphics/gmt`` and build. If you do not want default config settings, omit ``BATCH=YES``:
```

Comment:
If I understand it correctly, there is a Port recipe for GMT, and for each new GMT release, someone has to update the recipe. Am I right?

Reply:
OK. I find the recipes here https://svnweb.freebsd.org/ports/head/graphics/gmt/.

Reply:
It looks good! Please keep working on it.

Reply:
Correct! Someone has to update the port when there is a new release (the port maintainer). So I think it would be nice to have a section describing how to build it yourself as well. This is no major undertaking; it's almost exactly the same as on Debian/Ubuntu.